### PR TITLE
add code owner

### PR DESCRIPTION
**Pull request by kevinintel**
## Description

Add code owners

## Issues

n/a

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [Y] Documentation

## Dependencies

n/a

## Tests

n/a

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/authN-authZ/ ruoyu.ying@intel.com
+/helm-charts/ lianhao.lu@intel.com
```
Comment (lianhao) [inline .github/CODEOWNERS:2]:
```suggestion
/helm-charts/ dolpher.du@intel.com lianhao.lu@intel.com
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+/kubernetes-addons/ yingchun.guo@intel.com
+/microservices-connector/ huailong.zhang@intel.com lianhao.lu@intel.com
```
Comment (lianhao) [inline .github/CODEOWNERS:4]:
```suggestion
/microservices-connector/ huailong.zhang@intel.com kefei.zhang@intel.com
```

**Review by daisy-ycguo — APPROVED** [review]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/authN-authZ/ ruoyu.ying@intel.com
```
Comment (mkbhanda) [inline .github/CODEOWNERS:1]: add a first line that is the default, better matches down the file take precedence. Should we trouble Yingchun for everything?

*         yingchun.guo@intel.com
*.md      iris.ding@intel.com  malini.bhandaru@intel.com

**Review by mkbhanda — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 3 above)
Comment (mkbhanda) [reply .github/CODEOWNERS:1]: weird .. I had "*  yingchun.gua@intel.com"  but the GitHub system made it a bullet!!

Comment (kevinintel) [reply .github/CODEOWNERS:2]: I add yingchun,iris and malini as default owner
*.md may not CODEOWNER format

**Review by mkbhanda — APPROVED** [review]
LGTM

We just have one person for authn&z and scripts ,, for two reviewer rule

**Pull request merged**